work2/myworld/members/views.py

```diff
@@ -0,0 +1,43 @@
+from django.views import View
+from .models import Employees
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.decorators import method_decorator
+import pdb
+
+@method_decorator(csrf_exempt, name='dispatch')
+class EmployeeView(View):
+    def get(self, request, empid=None, department=None):
+        pdb.set_trace()
```

Comment:
@sampleharvgithub Why do you have pdb(debugger) enabled in your code?

Reply:
i used it to identify any issue in my code

Reply:
@sampleharvgithub I know that the purpose of the debugger is to identify the issues but please avoid having debugging code in your PR.